django/template/base.py

```diff
@@ -158,6 +158,17 @@ def loader_name(self):
                 self.loader.__module__, self.loader.__class__.__name__,
             )
 
+    def is_stale(self):
```

Comment:
I think using a `stale` property here would make more sense. It would avoid a function call in some case and would be more consistent with the current `Origin` API. e.g. `loader_name` uses `@property` instead of being called `get_loader_name()`.